src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -164,17 +166,9 @@ export class TemplateActionService extends TemplateInstanceService {
         const processor = new TemplateProcessService(this.injector);
         return processor.processTemplateWithoutRender(templateToProcess);
       case "emit":
-        const [emit_value, emit_from] = args;
```

Comment:
this was the previous incorrect implementation, that assumed all emit statements were of the format `emit:[emit_value]:[emit_from]`. Instead we will assume a more general `[emit_value]:[emit_data]` where data could represent any number of things depending on context